overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -100755,7 +100761,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "To actually make it work...",
-          "desc:8": "Craft a Shape Card: Quarry! Select two corners of an area to mine and insert it into the Builder! Power up the Builder and you can use it as a Quarry now!\n\nIf you don\u0027t want to destroy the environment, enter the Deep Dark and mine out that dimension instead! The Deep Dark Portal should be craftable now."
+          "desc:8": "Craft a Shape Card: Quarry! Select two corners of an area to mine and insert it into the Builder! You can also open the GUI for the shape card to configure the card as well. Power up the Builder and you can use it as a Quarry now!\n\nIf you don\u0027t want to destroy the environment, enter the Deep Dark and mine out that dimension instead! The Deep Dark Portal should be craftable now."
```

Comment:
could you describe how to properly open the GUI? iirc opening it via the button inside the builder doesnt save properly.

Reply:
Doing both in world and in the builder GUI works fine, I can add info for both though